src/cr/cube/matrix/cubemeasure.py

```diff
@@ -1042,6 +1036,24 @@ def _array_type_std_res(self, counts, total, rowsum, colsum):
         variance = rowsum * colsum * (total - rowsum) * (total - colsum) / total ** 3
         return (counts - expected_counts) / np.sqrt(variance)
 
+    @lazyproperty
+    def _column_is_array(self):
+        """Bool indicating whether the column is an array variable type
+
+        `_dimensions` is already apparent dimensions, so the columns is the last dimension.
+        (order is rows, column if 2D or tabs, rows, columns if 3D)
+        """
+        return self._dimensions[-1].dimension_type in DT.ARRAY_TYPES
+
+    @lazyproperty
+    def _row_is_array(self):
+        """Bool indicating whether the row is an array variable type
+
+        `_dimensions` is already apparent dimensions, so the rows is the 2nd to last
+        dimension. (order is rows, column if 2D or tabs, rows, columns if 3D)
+        """
+        return self._dimensions[-2].dimension_type in DT.ARRAY_TYPES
+
```

Comment:
I think you don't need 2 properties considering that these guards are used only 1 time, so you can use directly
`self._dimensions[-1].dimension_type in DT.ARRAY_TYPES` as condition in the row and col comparable counts.